### DISCUSSION TITLE
Horizontal Scroll

### DISCUSSION THREAD

**Discussion by jjboi8708** [noah-nuebling/mac-mouse-fix#504] [Questions] [answered]
How can I add the shift key to one of my mouse buttons so that whenever I want to scroll horizontally I can just hold one of my mouse buttons while scrolling with my scroll wheel? Whenever I add the shift key to the keyboard shortcut, it doesn't add the key. How can I fix this?

**Comment by noah-nuebling** [comment, marked as answer]
Hi @jjboi8708, scrolling horizontally by holding a mouse button is possible in the latest [MMF 3.0.0 Beta](https://github.com/noah-nuebling/mac-mouse-fix/releases/tag/3.0.0-Beta-6). Just assign "Click and Scroll" on any button to "Horizontal Scroll".

You can also use the new "Click and Drag" to "Scroll & Navigate" feature in 3.0.0 to scroll horizontally.

Let me know if you have any more questions!
